src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1480,6 +1482,12 @@ public DescriptorImpl() {
             load();
         }
 
+        @Nonnull
```

Comment:
No jsr-305 annotations, need to use spoptbugs instead.
```suggestion
        @NonNull
```